### ISSUE TITLE
Can't update Redmine : [...] activesupport [...] `require': No such file to load -- ../lib/redmine/Pagination (LoadError)

### ISSUE THREAD

**Issue by davidbouche** [dhanasingh/redmine_wktime#120]
Can update Redmine, here's the error :

```
2017-09-25T13:14:29.065Z: /home/************/vendor/bundle/ruby/2.3.0/gems/activesupport-4.2.7.1/lib/active_support/dependencies.rb:274:in `require': No such file to load -- ../lib/redmine/Pagination (LoadError)
It seems that I'm not the only person to have this problem : https://stackoverflow.com/questions/46194782/redmine-erpmine-plug-in-error
```

My redmine installation :

```
Environment:
  Redmine version                3.3.3.stable
  Ruby version                   2.3.3-p222 (2016-11-21) [x86_64-linux]
  Rails version                  4.2.7.1
  Environment                    production
  Database adapter               Mysql2
SCM:
  Subversion                     1.8.16
  Git                            2.13.3
  Filesystem                     
Redmine plugins:
  redmine_agile                  1.4.4
  redmine_contacts               4.1.1
  redmine_contacts_helpdesk      3.0.8
  redmine_wktime                 2.8
```

How can we solve this and update our Redmine instances ?

Thanks for your support,
David

**Comment by karthickmadhiazhagan** [comment]
Could you please provide version of ERPmine and Redmine you were trying to upgrade?

**Comment by davidbouche** [comment]
I'm only try to update to WkTime 3.0.

Thanks for your support,
David

**Comment by davidbouche** [comment]
for Redmine dev team, it comes from your plugin : https://www.redmine.org/issues/27072#change-81318

**Comment by karthickmadhiazhagan** [comment]
We have fixed this bug on ERPmine 3.0.1 . Could you please try with ERPmine 3.0.1

Thanks,
Karthick
